src/node_stat_watcher.cc

```diff
@@ -74,7 +69,12 @@ StatWatcher::StatWatcher(Environment* env, Local<Object> wrap)
 
 StatWatcher::~StatWatcher() {
   Stop();
```

Comment:
Now that I look at it... calling Stop() here is probably a minor (if harmless) bug as it in turn calls MakeWeak() when the destructor is almost certainly invoked from a weak callback.

You didn't introduce it so it's fine to leave it be for now but I figured I'd point it out.